include/zephyr/device.h

```diff
@@ -942,6 +948,9 @@ static inline bool z_impl_device_is_ready(const struct device *dev)
  * @param prio Initialization priority.
  */
 #define Z_DEVICE_INIT_ENTRY_DEFINE(node_id, dev_id, init_fn_, level, prio)     \
+	BUILD_ASSERT(IS_ENABLED(Z_DEVICE_LEVEL_##level),                       \
+		     "Invalid level: " STRINGIFY(level));                      \
```

Comment:
Maybe "Unsupported level for device drivers:"? I feel like "invalid level" suggests that like the name is wrong or something else.

Reply:
done